src/storage-preview/azext_storage_preview/azext_metadata.json

```diff
@@ -0,0 +1,4 @@
+{
+    "azext.minCliCoreVersion": "2.0.32.dev0",
```

Comment:
Ha?

Reply:
Talked to @derekbekoe about this, its worth limiting the version of the CLI as the extension features might be affected by core changes.